### PR TITLE
Update config.yml

### LINKED ISSUE #9: PR template missing

### PR DESCRIPTION
**Pull request by alexemdesgagne**
**Warning :** Before creating a pull request, make sure that it does not already exists in the [pull requests](../). Thank you.

## Description
Enhanced the config.yaml file.

## Fixes
Link all the related issues from the issue tracker
- Closes: #

## How has this been tested ?
(If possible) describe the steps to test these modifications.

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 blank_issues_enabled: false
-"""
+
 contact_links:
   - name: SONIA AUV website
     url: https://sonia.etsmtl.ca/
     about: Visit our website and feel free to ask any questions.
-"""
+
```
Comment (Kam0ss) [inline .github/ISSUE_TEMPLATE/config.yml]: The wiki is missing

**Review by Kam0ss — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (alexemdesgagne) [reply .github/ISSUE_TEMPLATE/config.yml]: Done in commit 05f598c010d5a67009b6d9a946abee833e9293aa

**Review by supertoto29 — APPROVED** [review]
LGTM!!

**Review by Kam0ss — APPROVED** [review]
LGTM

**Pull request merged**